### PR TITLE
ENH print user for the rate limit info

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by beckermr** [comment]
@isuruf Let's merge this in an hour

File: .travis_scripts/create_feedstocks.py [hunk 1]
```diff
@@ -83,6 +83,7 @@ def print_rate_limiting_info(gh):
     print("")
     print("GitHub API Rate Limit Info:")
     print("---------------------------")
+    print("user: ", gh.get_user().login)
```
Comment (isuruf) [inline .travis_scripts/create_feedstocks.py]: This would fail if there's no requests left.

Comment (beckermr) [reply .travis_scripts/create_feedstocks.py]: I was trying to be clever. Hold on.

Comment (beckermr) [reply .travis_scripts/create_feedstocks.py]: Alright this should be fixed.

**Comment by beckermr** [comment]
OK. Let's try it!

**Pull request merged**